pkg/metrics/metrics.go

```diff
@@ -89,7 +91,7 @@ func (m *Metrics) Run(servingAddress string) error {
 	return nil
 }
 
-func (m *Metrics) AddImage(namespace, pod, container, imageURL string, isLatest bool, currentVersion, latestVersion string) {
+func (m *Metrics) AddImage(namespace, pod, container, imageURL string, isLatest bool, currentVersion, latestVersion, os, arch string) {
```

Comment:
These func arguments are getting quite unwieldy. Can we create a new struct "metrics.Entry" which included all of these parameters. This will be the input to `AddImage`